proxies/go/go.mod

```diff
@@ -3,23 +3,30 @@ module github.com/devcyclehq/test-harness/proxies/go
 go 1.19
 
 require (
-	github.com/devcyclehq/go-server-sdk/v2 v2.9.5
+	github.com/devcyclehq/go-server-sdk/v2 v2.13.1-0.20240429135854-e299e6ccb33a
```

Comment:
whats with the stuff on the end of the version tag?

Reply:
this is the direct hash reference to the SSE branch to allow  using that property.

Reply:
we can't commit that can we?

Reply:
We can - it doesn't actually matter because the install script overrides it anyway.
This is just to ensure that the local build doesn't fail

Reply:
when running locally shouldnt you use the local linking version of the dependency that's commented out below?

Reply:
That's an option - but the way I was debugging is to ref to the existing git ref to prevent needing to run both at once and manually attach debuggers to both - this allows you to just run one debug session in IntelliJ/WebStorm - given you know that the test harness is the problem. 

It's just two approaches - but without this change the local code will fail and not build until we merge the SSE PR in Go

Reply:
ok just remember to update it after